### ISSUE TITLE
It is possible to move thumb out of range

### ISSUE THREAD

**Issue by WonderCsabo** [oli107/material-range-bar#23]
If i drag the thumb fast towards the edge of the screen, it is possible to move the thumb out of the range. I am using `pinRadius` `0dp` and `selectorSize` `16dp`.

Please see attached screenshot. Normally the min value is 0.

![screenshot_20151112-123210](https://cloud.githubusercontent.com/assets/480485/11117408/d54818fc-8939-11e5-955f-acdb4029ff67.png)

**Comment by WonderCsabo** [comment]
This also happens if i click the side of the thumb towards to end of the range.

**Comment by WonderCsabo** [comment]
@oli107 this only happens if `pinRadius="0dp"`.

**Comment by WonderCsabo** [comment]
@oli107 after some debugging the problems happens in `onActionUp`, but i could not find the exact line.

**Comment by WonderCsabo** [comment]
@oli107 sorry for pinging but can you check this issue?

**Comment by rameen** [comment]
yes its a bug.I have replicated it many times. There is no check for bounds.

**Comment by rmasarovic** [comment]
my dummy qucik fix workeround :
private RangeBar rangeBar;
private int min;
private int max;
 
rangeBar.setOnRangeBarChangeListener(new RangeBar.OnRangeBarChangeListener() {
            @Override
            public void onRangeChangeListener(RangeBar rangeBar, int leftPinIndex, int rightPinIndex,
                                              String leftPinValue, String rightPinValue) {
                int left = Integer.valueOf(leftPinValue);
                int right = Integer.valueOf(rightPinValue);
                if (left < min) {
                    rangeBar.setRangePinsByValue(min, right);
                }
                if (right > max) {
                    rangeBar.setRangePinsByValue(left, max);
                }
            }
